module1/lessons/index.md

```diff
@@ -36,7 +36,7 @@ title: Module 1 - Lessons
 
 ## Debugging
 * [Async Intro to Debugging](./debugging_prework)
-<!-- * [Debugging Techniques](./debugging_techniques) -->
+* [Debugging Techniques](./debugging_techniques)
```

Comment:
I think this line is linking this lesson to itself, right?  I would remove this line.

Reply:
@hfaerber this is the index file, which is a link to the lesson I made. If I remove it, then it won't show up on our curriculum site.